tests/Fixer/FunctionNotation/MethodArgumentSpaceFixerTest.php

```diff
@@ -134,59 +134,79 @@ public function testFixProvider()
                 '<?php function xyz(
                     $a=10,      //comment1
                     $b=20,      //comment2
-                    $c=30) {
+                    $c=30
+                ) {
                 }',
             ],
             'must keep align comments (2)' => [
                 '<?php function xyz(
                     $a=10,  //comment1
                     $b=2000,//comment2
-                    $c=30) {
+                    $c=30
+                ) {
                 }',
             ],
             'multiline comments also must be ignored (I)' => [
-                '<?php function xyz(
+                <<<'EXPECTED'
+<?php function xyz(
```

Comment:
I know it does, but I don't think that's really what's under test here. 

> I'm not sure why this string needs changing to nowdoc here?

It doesn't, it's just that I think nowdoc is better in this case, for various reasons, the first of which is : you are more sure of what you are testing. I bet the author of this meant to test

```
    function ()
    {
    }
```

and not

```
function ()
    {
    }
```